01-python/infrastructure/tree.py

```diff
@@ -1,3 +1,53 @@
-def draw_tree():
-    print("tree not found")
-    return
+def draw_tree(numTrees=0, printAxis="horizontal"):
+    """
+    Draws a tree(s).
+
+    Parameters:
+        numTrees: integer value; the number of trees to draw
+        printAxis: string value of "horizontal" or "vertical" (case insensitive); indicates whether to draw the tree(s)
+            horizontally or vertically 
+
+    Helper methods:
+        draw_top_of_tree: draws the top (i.e. leaves) of the tree numTrees times
+        draw_bottom_of_tree: draws the bottom (i.e. trunk) of the tree numTrees times
+
+    """
+
+    def draw_top_of_tree(numTrees):
+        for i in range(numTrees):
+            print("  $$  ", end="")
+        print()
+        for i in range(numTrees):
+            print(" $$$$ ", end="")
+        print()
+        for i in range(numTrees):
+            print("  $$  ", end="")
+        print()
+
+    def draw_bottom_of_tree(numTrees):
+        for i in range(numTrees):
+            print("  ||  ", end="")
+        print()
+
+    # Validation checks
+    validAxes = ["horizontal", "vertical"]
```

Comment:
Nit: can you capitalize this constant?
```suggestion
    VALIDAXES = ["horizontal", "vertical"]
```